### PR TITLE
Add vw and remove %, em option from font size in all blocks

### LINKED ISSUE #338: Add vw and remove %, em option in font size in all blocks
image

### PR DESCRIPTION
**Pull request by ishitagupta94**
Closes #338

### CODE REVIEW

**Review by Gaganbird — APPROVED** [review]

File: DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/button.js [hunk 1]
```diff
@@ -135,16 +135,11 @@
...
 						],
-						unitOptions: true,
-						units: [
-							{ name: 'px' },
-							{ name: '%' },
-							{ name: 'em' }
-						],
+                        units: [
+                            { name: 'px', min: 10, max: 100, step: 1, value: 16 },
+                            { name: 'vw', min: 0.5, max: 10, step: 0.1, value: 1 },
```
Comment (meetmandeep) [inline DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/button.js:140]: vw should be default or the 1st selected option. Please confirm.

File: DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/divider.js [hunk 2]
```diff
@@ -56,15 +56,11 @@
 						cssproperties: [
 							{ name: "width" }
 						],
-						unitOptions: true,
-						units: [
-                            { name: 'px' },
-                            { name: '%' }
-						],
+                        units: [
+                            { name: 'px', min: 10, max: 1000, step: 1, value: 100 },
```
Comment (meetmandeep) [inline DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/divider.js]: Why max 1000? Can't I create a divider that's 1980 px wide ?

**Review by meetmandeep — CHANGES_REQUESTED** [review]

File: DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/button.js (hunk 1 above)
Comment (ishitagupta94) [reply DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/button.js:140]: default changed to vw

File: DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/divider.js (hunk 2 above)
Comment (ishitagupta94) [reply DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/divider.js]: Updated divider max to 1980 px

File: DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/divider.js [hunk 3]
```diff
@@ -57,7 +57,7 @@ export default (editor, config = {}) => {
 							{ name: "width" }
 						],
                         units: [
-                            { name: 'px', min: 10, max: 1000, step: 1, value: 100 },
+							{ name: 'px', min: 10, max: 1980, step: 1, value: 100 },
```
Comment (meetmandeep) [inline DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/divider.js]: This should be 1920; not 1980. I'm not a designer :)

**Review by meetmandeep — CHANGES_REQUESTED** [review]

**Review by meetmandeep — APPROVED** [review]

**Pull request merged**